app/controllers/api/v1/planning_applications_controller.rb

```diff
@@ -67,21 +72,17 @@ def send_not_found_response
            status: :not_found
   end
 
-  def create_site
-    if site_params
-      site = Site.create_with(site_params)
-          .find_or_create_by!(uprn: site_params[:uprn])
-      site.id
-    end
-  end
-
 private
 
   def planning_application_params
     permitted_keys = %i[application_type
                         description
                         ward
-                        site_id
+                        address_1
+                        address_2
+                        town
+                        postcode
+                        uprn
```

Comment:
As you are setting these params directly (currently on line 35), they don't need to be permitted on the top level of the hash.